application/controllers/admin/useraction.php

```diff
@@ -186,6 +186,91 @@ public function adduser()
     }
 
     /**
+    * Send email to LimeSurvey selected users
+    *
+    */
+    function sendMailToUser()
```

Comment:
1. Should have public/private/protected.
2. No function should be longer than 60 lines, for readability. Please factor out in smaller functions.